src/app/pages/news/11.2/02-portal.md

```diff
@@ -1,10 +1,34 @@
 ## Axon Ivy Portal {#portal}
 
-Blah Portal blah.
+Introducing Axon Ivy Portal {#portal}! Packed with new features like notifications, enhancements to the Portal Dashboards, and multiple improvements for a smoother and more efficient user experience. 
```

Comment:
Do we need the first sentence at all? It is already clear from the title. Remove {#portal}

Reply:
What about statistics?

Reply:
As statistics is not ready yet by its full functionality, we won't go for that entry until it will be added later.

Reply:
currently it is still very buggy and only config by json, so we can use it in projects but i would marketing it publicly with 11.3 when we have the full ui support

Reply:
Removed first sentence